includes/functions.php

```diff
@@ -1432,17 +1432,29 @@ function cache_mac_oui()
     if ($lock->get()) {
         echo 'Caching Mac OUI' . PHP_EOL;
         try {
-            $mac_oui_url = 'https://macaddress.io/database/macaddress.io-db.json';
+            $mac_oui_url = 'https://raw.githubusercontent.com/wireshark/wireshark/master/manuf';
```

Comment:
https://gitlab.com/wireshark/wireshark/-/raw/master/manuf is the offical

Reply:
yep. Github is a readonly mirror as far as I can tell. Will change it.